src/modules/wishlists/wishlist.controller.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Param,
+  Delete,
+  NotFoundException,
+  UseGuards,
+  HttpCode,
+} from '@nestjs/common';
+import { WishlistService } from '#/modules/wishlists/wishlist.service';
+import { Wishlist } from '#/modules/wishlists/entities/wishlist.entity';
+import { GetAccount } from '../auth/decorators/get-account.decorator';
+import { AccessTokenGuard } from '#/shared/guards/access-token.guard';
+import { RolesGuard, UserRoles } from '#/shared/guards/roles.guard';
+import { Roles } from '../../shared/constants/user-roles.constants';
+
+@Controller('wishlists')
+export class WishlistController {
+  constructor(private readonly wishlistService: WishlistService) {}
+
+  @Post()
+  @UseGuards(AccessTokenGuard)
+  async create(
+    @GetAccount('accountId') accountId: string,
+    @Body('accommodationId') accommodationId: string,
+  ): Promise<Wishlist> {
+    return await this.wishlistService.create(accountId, accommodationId);
+  }
+
+  @Get()
+  @UserRoles(Roles.Admin)
+  @UseGuards(AccessTokenGuard, RolesGuard)
+  async findAll(): Promise<Wishlist[]> {
+    return await this.wishlistService.findAll();
```

Comment:
Let's add an endpoint for getting wishlist of user

Reply:
fixed